resources/lang/vi.json

```
@@ -51,5 +51,25 @@
     "Whoops! Something went wrong.": "Đã có lỗi xảy ra!",
     "Yes": "Đồng ý",
     "You are already subscribed.": "Bạn đã đăng kí.",
-    "The password must be at least 8 characters": "Mật khẩu phải ít nhất 8 ký tự."
+    "The password must be at least 8 characters": "Mật khẩu phải ít nhất 8 ký tự.",
```

```suggestion
    "password_validation": "Mật khẩu phải ít nhất 8 ký tự.",
```
M suggest đặt key ngắn gọn thôi nhé.